msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -42,19 +42,21 @@ Tabs allow the team members to access services and content in a specific space w
 
 With the pre-meeting app experience, you can find and add meeting apps. You can also do pre-meeting tasks, such as developing a poll to survey the meeting participants.
 
-To add tabs to an existing meeting:
+**To add tabs to an existing meeting**
 
 1. In your calendar, select a meeting to which you want to add a tab.
 1. Select the **Details** tab and select <img src="~/assets/images/apps-in-meetings/plusbutton.png" alt="Plus button" width="30"/>. The tab gallery appears.
 
-    :::image type="content" source="~/assets/images/apps-in-meetings/Pre-Meeting-002.png" alt-text="Pre-meeting app experience.":::
+    :::image type="content" source="~/assets/images/apps-in-meetings/PreMeeting.png" alt-text="Pre-meeting app experience.":::
 
 1. In the tab gallery, select the app that you want to add and follow the steps as required. The app is installed as a tab.
 
    > [!NOTE]
    >
    > * You can also add a tab to an existing meeting using the meeting **Chat** tab.
    > * Tab layout must be in an organized state, if there are more than 10 polls or surveys.
+   > * To support adding tabs in scheduled channel meetings, specify **team** scope in **scopes** section in your app manifest. Without **team** scope the app would not appear in the flyout for channel meetings.
```

Comment:
I don't think this quite makes sense in this location.... this section describes the *user* steps to add a tab to a meeting, but this bullet point (and the one above about "organized state") are about what the *developer* needs to do.

This specific bullet point sounds like it should be in the "Enable and configure" page.

The one about being organized if there are more than 10 polls or surveys... 🤔 feels like a non sequitur that we should just delete?

Reply:
@v-ksar please move this line in the notes for "Update your app manifest"

Reply:
remove second bullet point about "10 polls" and add a new note as "Adding an app to channel meeting installs the app for whole team. If the app supports bot, that will also be added at team level. You can configure tab at a per channel meeting level."

update "Tabs already in the channel are not automatically brought into the channel meeting." to "Tabs added to the channel are not brought into the channel meeting"

Reply:
Updated as mentioned.